tests/ballerina-compiler-api-test/src/test/java/io/ballerina/semantic/api/test/TypedescriptorTest.java

```diff
@@ -110,16 +109,12 @@ public void testConstantType() {
     @Test
     public void testFunctionType() {
         Symbol symbol = getSymbol(43, 12);
-        FunctionTypeDescriptor type = (FunctionTypeDescriptor) ((FunctionSymbol) symbol).typeDescriptor();
+        FunctionTypeDescriptor type = ((FunctionSymbol) symbol).typeDescriptor();
         assertEquals(type.kind(), TypeDescKind.FUNCTION);
 
-        List<Parameter> reqParams = type.requiredParams();
-        assertEquals(reqParams.size(), 1);
-        validateParam(reqParams.get(0), "x", REQUIRED, INT);
-
-        List<Parameter> defParams = type.defaultableParams();
-        assertEquals(defParams.size(), 1);
-        validateParam(defParams.get(0), "y", DEFAULTABLE, FLOAT);
```

Comment:
Shall we add this call as well?

Reply:
We don't have this API anymore. What if we extract the defaultable parameters and check?

Reply:
Modified